app/components/primer/octicon_symbol_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" hidden>
```

Comment:
I didn't see any a11y concerns locally with adding `hidden` here, but would be curious to know if there's something I'm missing.

Reply:
@khiga8 👀

Reply:
whoops sorry I missed this! 

It looks like there's pretty decent support for `hidden` now based on [this][0]. 
But... the hidden attribute can easily be overridden if there's CSS setting the `display`. For more reliable support I might consider  using `style="display: none;"`

[0]: https://www.tpgi.com/the-state-of-hidden-content-support-in-2016/

Sources: 
https://www.tpgi.com/the-state-of-hidden-content-support-in-2016/
https://css-tricks.com/the-hidden-attribute-is-visibly-weak/
https://meowni.ca/hidden.is.a.lie.html

Reply:
Side note: It looks like there's [no need] for `<defs>` given we're rendering `<symbol>` !

[no need]: https://css-tricks.com/svg-symbol-good-choice-icons/#why-symbol-is-better-for-icons

Reply:
We do have a selector in primer for `[hidden]` to set it display none I think Monica suggests in her post. https://github.com/primer/css/blob/63764f7edd59c5f4949c91e5373471aa8b0d4865/src/base/normalize.scss#L84